app/assets/javascripts/error_handler.js.coffee

```
@@ -0,0 +1,11 @@
+# Initialize Costanza front-end error handling/tracking
+Costanza.init (info, rawError) ->
+  $.ajax
+    url: 'application/client_error'
+    method: 'POST'
+    dataType: 'json'
+    data: info
+    success: (message) ->
+      bonnie.showError(message) if message
+    error: () ->
+      console.error(info) if info
```

This completely eats the error as far as the user is concerned. Perhaps the bonnie.showError() should be called to let them know there was an error but an issue with Bonnie or their connection prevented from more details being available.